invenio_app_ils/ui/src/invenio_app_ils/pages/backoffice/UserDetails/components/ItemsSearch/ItemsSearch.js

```diff
@@ -127,24 +127,25 @@ export default class ItemsSearch extends Component {
     return (
       <Grid columns={3} verticalAlign="middle" stackable relaxed>
         <Grid.Column width={5} textAlign="left">
-          <div>{totalResults} results</div>
+          <div>Found {totalResults} item(s).</div>
         </Grid.Column>
       </Grid>
     );
   };
 
   render() {
-    const { items, isLoading, hasError, queryString, error } = this.props;
+    const { items, isLoading, queryString, error } = this.props;
     return (
       <Segment>
         <Header as={'h3'}>Items</Header>
+        <Header.Subheader>Search items by barcode.</Header.Subheader>
         <Container>{this._renderSearchBar()}</Container>
         <Grid columns={1} stackable relaxed className="items-search-container">
           <Grid.Column width={16}>
             <Loader isLoading={isLoading}>
               <Error error={error}>
-                {this._renderHeader(items.length)}
-                {!(_isEmpty(items) && !hasError)
+                {!_isEmpty(items) && this._renderHeader(items.hits.length)}
```

Comment:
Maybe we can add the header inside the relevant component that will be rendered. EmptyList or ResultList.

I have also noticed that Empty list is wrapped in a `<Segment>` while ResultList in a plain `<div>`. I would use the same element, `segment` for all.